docker-compose.yml

```diff
@@ -32,24 +32,22 @@ services:
       - ./docker/app/storage:/var/www/app/storage:rw,delegated
     depends_on:
       - db
-    networks: 
-      - invoiceninja  
+    networks:
+      - invoiceninja
     extra_hosts:
       - "in5.localhost:192.168.0.124 " #host and ip
 
   db:
     image: mysql:5
+#    When running on ARM64 use MariaDB instead of MySQL
+#    image: mariadb:10.4
 #    For auto DB backups comment out image and use the build block below
 #    build:
 #      context: ./config/mysql
     ports:
       - "3305:3306"
     restart: always
-    environment: 
```

Comment:
Can you retain these variables but change the values to point to env vars like this: 

https://github.com/invoiceninja/dockerfiles/blob/1d53d2cbb205f831d0a22448f94e5bd04ad4d690/docker-compose.yml#L51

Include MARIA_USER and so on uncommented below. That way there’s no need to worry about switching between MARIA and MYSQL env.

For the ROOT_PASSWORD can you reference ${DB_ROOT_PASSWORD} instead

Reply:
@lwj5, thanks for the remark, but I'm unsure I understand you well 😕 

If I get you well, your are talking about including (uncommented) `MARIA_***` values in the `docker-compose.yml` file. Why would that be?\
If one refers to the [dockerhub.com MariaDB page](https://hub.docker.com/_/mariadb), it is clearly indicated under the **Environment Variables** section, that these variables are **_strictly identical_** to the ones used by the [MySQL docker image](https://hub.docker.com/_/mysql), i.e.: the MariaDB image makes use of the exact same environment variables as would the MySQL image, I cannot see the point in adding (unused) `MARIA_***` values.

So when you say:
> That way there’s no need to worry about switching between MARIA and MYSQL env.

It is in fact **already the case** with the proposed configuration, since MariaDB is _willingly trying to be as MySQL compatible as possible_.

Regarding your remark saying:
> Can you retain these variables but change the values to point to env vars

This might indeed bring a bit more clarity to the environment variables declaration, and to be honest I myself found it a bit disturbing declaring some values twice but didn't think of referencing them using `${VAR_NAME}` 😉 

As soon as I get your feedback regarding your proposed `MARIA_***` values, I'll update my PR accordingly...

Reply:
From #307 

> I've checked both MySQL's and MariaDB's dockerhub pages and I can confirm that both Environment Variables and Docker Secrets sections are a copy/paste of each others with only the term mysql replaced by mariadb.

i understood from you that the prefixes has to be replaced. I went to check on docker hub and it seems it’s not the case so all is good.

Reply:
@lwj5, thank you for the merge!
> i understood from you that the prefixes has to be replaced. I went to check on docker hub and it seems it’s not the case so all is good.

Oh, yes sorry, I understand your interpretation of my comment, I'll try to be more explicit next time 😉